### DISCUSSION TITLE
Need help on how to put mods in pc :(

### DISCUSSION THREAD

**Discussion by Qinnya** [thatskymod/Sky-CotL-Scripts#134] [Q&A] [answered]
Hi! I'm new this kind of stuff about pc because i always use my phone, yes i use mod on my phone.

I'm kind of struggling how to do it on pc, please someone tell me how from the beginning step by step🙏

![image](https://github.com/user-attachments/assets/421bb421-d948-4a99-a7b8-74249d7b18b6)

**Comment by thatskymod** [comment]
Hello!

Consider looking into **ThatSkyApp** by @XeTrinityz — an automated installation assistant for the mod known as **ThatSkyMod**: https://github.com/XeTrinityz/ThatSkyMod/releases/latest

In particular, check out the "Automated Installation Instructions":
> 1. Download `ThatSkyApp.exe` from this release and run it.  
> 2. Click Install (If a folder selection dialog appears, select your game folder).  
> 3. Click `Launch Sky`.

With this tool, you don’t have to worry about installing dependencies or remembering where to place the mod files, as the program takes care of everything by itself.

**Reply by Qinnya** [reply]
Hello thank you for replying, so I did what I can do and this is that only thing came up. I'm really not good with this stuff

![image](https://github.com/user-attachments/assets/0d116e7d-a3c6-44d6-8f04-cfbbf14a572d)

**Comment by thatskymod** [comment]
Sorry for the trouble, and thanks for sharing the screenshot!

The black box showing '`TSM initialized`' and '`Searching for TgcMainWindow`' gives helpful insight into what's happening. At this stage, Sky should typically be starting or do so within a few seconds at most. Therefore, as far as the mod installation is concerned, you’ve done everything correctly. If Sky doesn’t start automatically, there may be an issue with game file integrity or with resource allocation (though as long as your PC meets Sky’s minimum requirements, the latter is less likely).

~Let me see if I can pinpoint the cause.~ (Update: Unfortunately, I wasn't able to reproduce the exact scenario).

Meanwhile the community is welcome to jump in with any further insights or suggestions!

**Comment by thatskymod** [comment, marked as answer]
Hello, quick follow-up: You can try to "Repair" the Installation using the newly released ThatSkyApp v1.6.0. This would check and fix the integrity of corrupted files if any such files are present.
